### PR TITLE
Set Windows console codepage to Unicode

### PR DESCRIPTION
**Pull request by jmyersmsft**
@ericsciple

### CODE REVIEW

File: Tasks/NuGetInstaller/nugetinstaller.ts [hunk 1]
```diff
@@ -33,6 +33,11 @@ async function main(): Promise<void> {
     try {
         tl.setResourcePath(path.join(__dirname, "task.json"));
 
+        // set the console code page to "Unicode"
+        if (process.platform === "win32") {
```
Comment (bryanmacfarlane) [inline Tasks/NuGetInstaller/nugetinstaller.ts]: This looks good.  I think we should consider doing the same change inside toolrunner.exec.  Eric and I have been discussing.  If you want to do a PR there if you agree, we can take that.

Comment (bryanmacfarlane) [inline Tasks/NuGetInstaller/nugetinstaller.ts]: I believe 65001 is utf-8 so update the comment

Comment (jmyersmsft) [inline Tasks/NuGetInstaller/nugetinstaller.ts]: fixed

**Comment by jascam** [comment]
:shipit:

**Pull request merged**